camel-k-loader-groovy/src/test/resources/routes-with-components-configuration.groovy

```diff
@@ -28,7 +28,7 @@ context {
 
         mySeda(SedaComponent) {
             // set value as method
```

Comment:
I assume the comment above is no longer correct

Reply:
fixed